Twig/Extension/ArrayExtension.php

```diff
@@ -41,6 +42,7 @@ public function mapBy(array $input, $key)
                 return $item[$key];
             }
 
+            // TODO: use PropertyAccessor ??
```

Comment:
It's not mandatory, just a proposition.
If I well understand, we are here doing like the "PropertyAccessor", why don't we use it?